broker_app.py

```diff
@@ -25,6 +25,7 @@
 logging.getLogger('ingest').setLevel(logging.INFO)
 logging.getLogger('ingest.api.ingestapi').setLevel(logging.INFO)
 logging.getLogger('broker.service.spreadsheet_upload_service').setLevel(logging.INFO)
+logging.getLogger('broker.submissions.export_to_spreadsheet_service').setLevel(logging.INFO)
```

Comment:
Please note that I've only moved all the routes inside create_app() to avoid ingest API being called everytime  app in broker_app.py is being imported in tests